src/main/java/it/gov/pagopa/pu/organization/enums/PagoPaInteractionModel.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.pu.organization.enums;
+
+public enum PagoPaInteractionModel {
+  SYNC,
+  SYNC_ACA,
+  SYNC_GPDPRELOAD,
+  SYNC_ACA_GPDPRELOAD,
```

Comment:
the `$` sign has been used as separator
```suggestion
  SYNC$GPDPRELOAD,
  SYNC_ACA$GPDPRELOAD,
```